### PR TITLE
Use the general null handling in array_remove

### PR DESCRIPTION
**Pull request by zacw7**

### CODE REVIEW

**Comment by zacw7** [comment]
https://github.com/facebookincubator/velox/issues/6349

File: velox/functions/prestosql/ArrayFunctions.h [hunk 1]
```diff
@@ -727,7 +727,7 @@ struct ArrayRemoveFunction {
       const arg_type<Array<Generic<T1>>>& array,
       const arg_type<Generic<T1>>& element) {
     static constexpr CompareFlags kFlags = {
```
Comment (xiaoxmeng) [inline velox/functions/prestosql/ArrayFunctions.h:729]: @zacw7  Can we add test to catch the difference in null handling? Thanks!

**Review by xiaoxmeng — COMMENTED** [review]
@zacw7 LGTM. Thanks!

**Comment by zacw7** [comment]
Fixed in https://github.com/facebookincubator/velox/pull/6424

**Pull request closed without merging**